org.OpenT2T.Sample.SuperPopular.Thermostat/Nest Thermostat/js/README.md

```diff
@@ -0,0 +1,92 @@
+access_token : 
```

Comment:
Is the first part of this README required? lines 1-11? Not sure if you want to check in your access token like that either (somebody will be able to use it to mess with your thermostat)